api-docs/cloud-serverless/ref.yml

```diff
@@ -14002,26 +14011,31 @@ paths:
           - [Best practices for optimizing writes](/influxdb/cloud-serverless/write-data/best-practices/optimize-writes/)
         required: true
       responses:
-        '204':
+        "204":
           description: |
-            Success. Data is written and queryable.
-        '400':
+            Success. All points in the batch are written and queryable.
+        "400":
```

Comment:
201 when partial-writes are on, and some lines were written.
Otherwise, still a 400.